unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs

```diff
@@ -134,11 +136,32 @@ private void View_OnFavoritesClicked(string placeUUID, bool isFavorite)
 
     private void FirstLoading()
     {
-        view.OnPlacesSubSectionEnable += RequestAllPlaces;
-        view.OnFilterSorterChanged += RequestAllPlaces;
+        view.OnPlacesSubSectionEnable += OpenTab;
+        view.OnFilterChanged += ApplyFilters;
+        view.OnSortingChanged += ApplySorting;
         cardsReloader.Initialize();
     }
 
+    private void OpenTab()
+    {
+        exploreV2Analytics.SendPlacesTabOpen();
+        RequestAllPlaces();
+    }
+
+    private void ApplyFilters()
+    {
+        if (!string.IsNullOrEmpty(view.filter))
```

Comment:
You are not sending the analytic if the filters have been cleared; if it wasnt a requirement, you can dismiss this comment.